token/cli/src/main.rs

```diff
@@ -324,51 +328,66 @@ pub fn signers_of(
     }
 }
 
+fn token_client_from_config(
+    config: &Config<'_>,
+    token_pubkey: &Pubkey,
+) -> Token<ProgramRpcClientSendTransaction> {
+    let token = Token::new(
+        config.program_client.clone(),
+        &config.program_id,
```

Comment:
Careful, this might not always be correct -- it defaults to `Tokenkeg...` but almost all of the commands figure out the correct program id to use based on the mint. The only exception is `create_token`, where we can't figure it out